mission_manager/src/behaviors/altitude_heading.cpp

```diff
@@ -34,22 +34,34 @@
 
 // Original version: Christopher Scianna Christopher.Scianna@us.QinetiQ.com
 
-#include "behaviors/altitude_heading.h"
+/*      Behavioral - Altitude Heading
+
+        <altitude_heading>
+            <description>
+                00:00:00 - .
+            </description>
+            <when unit="sec">0</when>
+            <timeout unit="sec">50</timeout>
+            <altitude unit="deg">10.0</altitude>
+            <heading unit="deg">180.0</heading>
+            <speed_knots>0.0</speed_knots>
+        </altitude_heading>
```

Comment:
@patapufate meta: I think this is relevant documentation that should be public. That is, it should be in the corresponding header file.

Reply:
Done